### ISSUE TITLE
no icons over buttons

### ISSUE THREAD

**Issue by EugeneKiver** [Harvie/DokuWiki-Plugin-SVGEdit#19]
I've installed svg-edit though plugin manager, added new image {{svg>...}} then created new page based on it, and when i press SVG button, interface appears, but all the buttons are blank and unresponsive. I tried to do the same in Chrome (I use Firefox as default) but it looks the same
![svg-edit-blank](https://cloud.githubusercontent.com/assets/4741934/6892420/a9300f56-d6e5-11e4-99ec-bde52b5d4bc9.jpg)

**Comment by EugeneKiver** [comment]
I've updated java and checked on both firefox and chrome again, but the problem persists

**Comment by dmitrylukin** [comment]
Hello, Eugene
Find file script.js and change 
*var svgeditor_path = 'https://svg-edit.googlecode.com/svn/branches/stable/editor/';   //online stable
//var svgeditor_path = 'https://svg-edit.googlecode.com/svn/trunk/editor/';       //online latest (unstable)
//var svgeditor_path = DOKU_BASE+'lib/plugins/svgedit/svg-edit/';               //offline*
to 
*//var svgeditor_path = 'https://svg-edit.googlecode.com/svn/branches/stable/editor/';   //online stable
var svgeditor_path = 'https://svg-edit.googlecode.com/svn/trunk/editor/';       //online latest (unstable)
//var svgeditor_path = DOKU_BASE+'lib/plugins/svgedit/svg-edit/';               //offline*

**Comment by Harvie** [comment]
BTW maybe you can try to use github URLs as google code is closed...
I believe new URLs are:
https://raw.githubusercontent.com/SVG-Edit/svgedit/stable/editor/
https://raw.githubusercontent.com/SVG-Edit/svgedit/master/editor/

BTW when i had the problem with incons it was caused by misconfigured mime-types on webserver (lighttpd).

**Comment by Harvie** [comment]
is this fixed now?

**Comment by konstruktur** [comment]
Hello!
I just installed svgedit on a an installation of DokuWiki (2017-02-19b "Frusterick Manners") and when I follow the instructions I have exactly the same editor view as @EugeneKiver "blank and unresponsive" (s. issue) - on both Chrome 58.0.3029.110 (64-bit) and Firefox (53.0.3 (32-Bit)).

![2017-05-23 19_19_13_dokuwiki_svgedit](https://cloud.githubusercontent.com/assets/8482709/26370380/e9a5d3f4-3ff7-11e7-8ee2-4c360e9bc9b8.png)

@dmitrylukin: I changed the script like proposed, but that did not fix it. 

@EugeneKiver: (How) Could you finally fix this issue?

What further infos should I provide here? (pagesource, server, ... ?) Thanks!

**Comment by Harvie** [comment]
I have the same problem, chromium says this:

embedapi.js:150 Uncaught DOMException: Blocked a frame with origin "https://wiki.spoje.net" from accessing a cross-origin frame.
    at https://cdn.rawgit.com/SVG-Edit/svgedit/stable/editor/embedapi.js:150:64

but everything seems to be set up correctly and crossorigin should be enabled on rawgit cdn. however it's still not working.

**Comment by konstruktur** [comment]
I went for the offline solution and 
1) uploaded the latest version of SVG-Edit (s. https://github.com/SVG-Edit/svgedit) to lib/plugins/svgedit/svg-edit and 
2) changed the script.js of svgedit accordingly (uncomment var svgeditor_path = DOKU_BASE+'lib/plugins/svgedit/svg-edit). 
3) Cleaned the browser cache and reloaded.

Now it works perfectly! 
Thanks for this plugin - great work!

**Comment by Harvie** [comment]
@konstruktur that surely should work, however i am not really happy with that solution. i want both modes to work... local install for privacy and cloud hosted approach for low-maitenance and upgrade-less operation.

**Comment by konstruktur** [comment]
@Harvie Yes, absolutely - I think in this case I can only live with no/manual updates, since the development of SVG-Edit (https://github.com/SVG-Edit/svgedit) seems not too active anymore - but that's no satisfiying solution. Ideas, anybody? Drawing SVGs in a DokuWiki is absolutely great, especially because of the inbuilt version-control - spread the word! I hope somebody can come up with a solution here.

**Comment by Harvie** [comment]
I've got idea! Maybe i can include few lines of reverse proxy script that will download and serve svgedit on the fly.
